diagnostic.md

```diff
@@ -76,7 +74,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+the string 'a stick of butter'
```

Comment:
Correct. Since you have a stick of butter in '', you don't need to specify that it is a string. This is implied by your response.